### PR TITLE
include instructions for dev testing

### PR DESCRIPTION
**Pull request by philschatz**
This adds instructions for testing a Pull Request.

refs #5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,16 @@ One of the most important things when starting out in any software project(s) is
...
+```
 
 Follow the instruction prompts given by the script
 
+## Development testing
+
+To test the code found in a Pull Request, determine the branch name for the Pull Request and then replace `${BRANCH_NAME}` in the command below:
+
+```sh
+bash <(curl -s https://raw.githubusercontent.com/openstax/setup-openstax-machine/master/${BRANCH_NAME}/setup.sh)
```
Comment (kerwinso) [inline README.md]: Shouldn't the `master/` be removed from this line?

**Review by kerwinso — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kerwinso) [reply README.md]: Oh and add the `/bin/`:
```sh
bash <(curl -s https://raw.githubusercontent.com/openstax/setup-openstax-machine/${BRANCH_NAME}/bin/setup.sh)
```

**Review by kerwinso — APPROVED** [review]

**Review by kerwinso — COMMENTED** [review]
how do I unapprove changes? Need to consider which branch the ansible playbook is running from.

**Comment by philschatz** [comment]
Ah, ansible clones the repository and uses the `master` branch. So it seems like the instructions would need to be updated to something like `BRANCH_NAME=foo bash <(curl -s https://raw.githubusercontent.com/openstax/setup-openstax-machine/${BRANCH_NAME}/bin/setup.sh)` and then maybe somehow change the `git clone` part of `setup.sh` to use `BRANCH_NAME` if it is available...


Also, @kerwinso you can click `[Reviewers]` and then select yourself, and then click `Files Changed`, and then `Reject`.

**Comment by m1yag1** [comment]
Hmm, I'm still wondering if this will be effective or not. No matter what the master branch is cloned down for the playbook. We would need to add a step that it is cloned down and then the proper branch is checked out. You can refer to this: https://github.com/openstax/setup-openstax-machine/blob/master/bin/setup.sh#L10-L12

**Review by kerwinso — CHANGES_REQUESTED** [review]
ah that feels good :P

**Comment by philschatz** [comment]
ok, the conclusion is to add a list of dev instructions in order to keep less branching code in the actual install script

**Comment by kerwinso** [comment]
Here's the README I'm testing against: https://github.com/openstax/setup-openstax-machine/tree/atom-plugins

I think the only step missing from the above file is to checkout the test branch (before installing dependencies).

<blockquote><div> GitHub</div><div><strong><a href="https://github.com/openstax/setup-openstax-machine">openstax/setup-openstax-machine</a></strong></div><div>Contribute to setup-openstax-machine development by creating an account on GitHub.</div></blockquote>

**Comment by m1yag1** [comment]
Phil you may want to rebase off of master as there has been updates ;)

**Review by m1yag1 — APPROVED** [review]

**Review by kerwinso — APPROVED** [review]

**Comment by philschatz** [comment]
Everyone approved 🎉  @m1yag1 can this be merged?

**Pull request merged**